src/components/CallView/CallView.vue

```diff
@@ -322,13 +322,17 @@ export default {
 		},
 
 		shouldShowPresenterOverlay() {
-			return this.shownRemoteScreenCallParticipantModel.attributes.videoAvailable || this.isModelWithVideo(this.shownRemoteScreenCallParticipantModel)
+			return this.shownRemoteScreenCallParticipantModel?.attributes?.videoAvailable || this.isModelWithVideo(this.shownRemoteScreenCallParticipantModel)
```

Comment:
Can you confirm that:
1. This is expected `shownRemoteScreenCallParticipantModel` to be nullish here (in which case)
2. Even with `shownRemoteScreenCallParticipantModel`, it is possible that `shownRemoteScreenCallParticipantModel.attributes` is nullish

Reply:
```js
		shownRemoteScreenCallParticipantModel() {
			if (!this.shownRemoteScreenPeerId) {
				return null
			}
			return this.callParticipantModels.find(callParticipantModel => {
				return callParticipantModel.attributes.peerId === this.shownRemoteScreenPeerId
			})
		},
```

1. It can be nullish when no other user screensharing.
2. Well, here it shouldn't be nullish, just added it to avoid extra exception. If it is null or undefined, it will be shown in `shownRemoteScreenCallParticipantModel` already.

Reply:
> just added it to avoid extra exception

We should avoid using optional chaining for such purpose. While it "removes" the error from the console and an exception from the function, it does not fix the real issue if any. It only hides it.

If there is a high level issue, it will be harder to find it in future, because instead of an exception for a specific line, we have everything working, just incorrect in further computing.

If there is no issues, then additional check is just useless, adding cognitive complexity.

Noting that, because we have a lot juch optional changing usage.